feat(i18n-HU): Update Hungarian translation

AntoszHUN opened the pull request:
Update gcode captions.

Signed-off-by: AntoszHUN <zsikaiantal@gmail.com>

pedrolamas commented on src/locales/hu.yaml:
```diff
@@ -1,508 +1,499 @@
+      mesh_matrix: Hálós mátrix
+      probed_matrix: Próbamátrix
+      profile_name: Profil Név
+      remove_profile: %{name} profil eltávolítása
```
The single quote `'` around this translation needs to be kept as this starts with a special replacement sequence.

pedrolamas commented on src/locales/hu.yaml:
```diff
@@ -1,508 +1,499 @@
+      flip_layout: Fordított elrendezés
+      hide_temp_waits: Hőmérsékletek elrejtése
+    placeholder:
+      command: Az automatikus kiegészítéshez használd a 'tab'-ot, parancslistához
```
As this is a multiline translation, this line needs to keep the `>-` and then put the rest of the translation on the next line indented by 2 spaces (as it was before the change)

pedrolamas commented on src/locales/hu.yaml:
```diff
@@ -1,508 +1,499 @@
+      pause: Szünet
+      preheat: Előfűtés
+      presets: Előbeállítások
+      preview_gcode: A G-Kód előnézete
```
Up to you on this one, but normally I wouldn't expect for "G-Code" to be translated as it is a specific technical term/format.

matmen replied:
I think having this as "G-Code" makes sense, the new chinese translation PR also uses the english version and IIRC so does the german translation

pedrolamas commented on src/locales/hu.yaml:
```diff
@@ -1,508 +1,499 @@
+      iframe: HTTP oldal
+    camera_rotate_options:
+      none: Nincs
+      90: 90°
+      180: 180°
+      270: 270°
```
The keys need to be surrounded by single quotes `'`, just as it was before the change!

pedrolamas reviewed (changes requested):
You need to keep the line endings on the file as LF but they have been changed to CRLF...

Whatever editor you are using to make these changes, the line endings must be kept as LF (Unix style) instead of CRLF (Windows style).

pedrolamas commented:
I'm on Windows too!

My advice, use [Visual Studio Code](https://code.visualstudio.com/) and install these 2 extensions:

- [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) - this will ensure you use the same formats as we need to
- [i18n Ally](https://marketplace.visualstudio.com/items?itemName=Lokalise.i18n-ally) - helper extension for translations!
![image](https://user-images.githubusercontent.com/85504/164196661-0d506438-3251-48a7-bcc6-6a1ddb823e94.png)

AntoszHUN commented:
I fixed the bugs. Can it be good now?

pedrolamas commented:
> I fixed the bugs. Can it be good now?

The line endings were still incorrect, but I fixed that for you on the PR (assuming you don't push more updates here, we should be good to merge it)

AntoszHUN commented:
> 



> > I fixed the bugs. Can it be good now?
> 
> The line endings were still incorrect, but I fixed that for you on the PR (assuming you don't push more updates here, we should be good to merge it)

All right. Thanks! I won't update then.

matmen commented:
ah, my bad, it should be `Gcode` not `G-Code`, will fix that

AntoszHUN commented:
> 



> ah, my bad, it should be `Gcode` not `G-Code`, will fix that

Why did I have to rewrite it? In English, G-Code means something. The Hungarian free translation of this is G-Code. Code = Kód. Why wasn't it good now? I translated the klipper3d.org website into Hungarian in 2.5 months. Everywhere the G-Code was rewritten into Hungarian G-Kód. No one fixed it because it wasn't a bug.

matmen commented:
From what I've seen so far fluidd's convention here is to stick to the english locale for "Gcode" - as you can see [here](https://github.com/fluidd-core/fluidd/blob/develop/src/locales/cn.yaml#L380) and [here](https://github.com/fluidd-core/fluidd/blob/develop/src/locales/de.yaml#L147) for example. I think it makes sense to stick to this, as I think it's more universal.

AntoszHUN commented:
> 

The translation does not change its meaning.

matmen commented:
I am aware of that, but fluidd's localization convention is to leave it untranslated. It's untranslated in all the other translations too.

AntoszHUN commented:
> I'm on Windows too!
> 
> My advice, use [Visual Studio Code](https://code.visualstudio.com/) and install these 2 extensions:
> 
>     * [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) - this will ensure you use the same formats as we need to
> 
>     * [i18n Ally](https://marketplace.visualstudio.com/items?itemName=Lokalise.i18n-ally) - helper extension for translations!
>       ![image](https://user-images.githubusercontent.com/85504/164196661-0d506438-3251-48a7-bcc6-6a1ddb823e94.png)

I installed the programs. But for the i18N Ally, how can it be made to work? He's not doing anything to me.

pedrolamas commented:
if you open the repo (folder) in Visual Studio, it should just detect the translations!
